client/lib/feed-post-store/post-fetcher.js

```diff
@@ -23,15 +23,15 @@ function PostFetcher( options ) {
 assign( PostFetcher.prototype, {
 
 	add: function( postKey ) {
-		this.postsToFetch = this.postsToFetch.add( Immutable.fromJS( omit( postKey, 'localMoment' ) ) );
```

Comment:
Whoa. It may be worth it to hone focus on removing immutable.  We only have 16 files that reference it!